mesh/mesh_readers.cpp

```diff
@@ -2030,6 +2030,35 @@ void Mesh::ReadGmshMesh(std::istream &input, int &curved, int &read_gf)
             getline(input, buff); // Read end-of-line
          }
 
+         // Follow existing long chains of slave->master in v2v array. Finally,
```

Comment:
Rather than "Finally, ..." it may be more clear to say "Upon completion of this loop each v2v[slave] will point to a true master vertex."